namespace cpp infra 
namespace java infra.gen

const string DATASPHERES_ROOT		        	= "dataspheres"
const string NODES_ROOT 		            	= "nodes"
const string SERVICES_ROOT		            	= "services"
const string DATOM_ROOT                             	= "/datom"
const string CONFIGSERVICE_ROOT                     	= "/datom/configservice"
const string DATASPHERE_ROOT_PATH_FORMAT            	= "/dataspheres/{}"
const string SERVICES_ROOT_PATH_FORMAT              	= "/dataspheres/{}/services"
const string SERVICE_ROOT_PATH_FORMAT               	= "/dataspheres/{}/services/{}"
const string REPLICA_CLUSTER_ROOT_PATH_FORMAT		= "/dataspheres/{}/replicacluster/{}"
const string REPLICA_CLUSTER_RING_ROOT_PATH_FORMAT	= "/dataspheres/{}/replicacluster/{}/rings/{}"
const string REPLICA_CLUSTER_RESOURCE_ROOT_PATH_FORMAT	= "/dataspheres/{}/replicacluster/{}/resources/{}"

const string TOPIC_SERVICES                         	= "services"
